test/Common/OrchestrationVersionTests.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
+{
+    public class OrchestrationVersionTests
+    {
+        private readonly ITestOutputHelper output;
+        private readonly TestLoggerProvider loggerProvider;
+
+        public OrchestrationVersionTests(ITestOutputHelper output)
+        {
+            this.output = output;
+            this.loggerProvider = new TestLoggerProvider(output);
+        }
+
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        [InlineData(null, "null")]
```

Comment:
Sorry I am a little bit confused here, I saw your reply below, but what will be the actual version value when cx define null, "", "null" in real cases?

Reply:
In the orchestrator function code, `ctx.Version` will contain exactly what the `DefaultVersion` in `host.json` contains (or `null` if it is not specified). In this test, I want to confirm that this is indeed the case, so that, for example, nulls are not silently replaced by empty strings or the other way around. I want to do it by looking at the string returned from the `GetOrchestrationVersion_AfterExternalEvent` orchestrator, so I need a way to reliably distinguish between `null`, an empty string, a string containing `"null"`, etc.

But I can see that it's causing more confusion than I anticipated, so I'll do something to make it clearer.

Reply:
Ok, I'm serializing this value to JSON now, I hope it clarifies things.